common/app/model/Cached.scala

```diff
@@ -29,6 +29,7 @@ object CacheTime {
   object DiscussionDefault extends CacheTime(60)
   object DiscussionClosed extends CacheTime(60, Some(longCacheTime))
   object Football extends CacheTime(10)
+  object Cricket extends CacheTime(10)
```

Comment:
I think existing cache time for the cricket endpoint is 60 seconds. Maybe 10 might be more suitable, but for now shall we leave it as it is for production and we can discuss it with the team to see if we need to change it?